osf/management/commands/update_preprint_share_dates.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import unicode_literals
+import logging
+
+from django.core.management.base import BaseCommand
+
+from scripts import utils as script_utils
+from osf.models import PreprintService
+from website.preprints.tasks import on_preprint_updated
+
+logger = logging.getLogger(__name__)
+
+def update_share_preprint_modified_dates(dry_run=False):
+    dates_updated = 0
+    for preprint in PreprintService.objects.filter():
```

Comment:
Rather than filter in Python, you can do a self-referential query using `django.db.models.F`. Something like

```python
PreprintService.objects.filter(date_modified__lt=F('node__date_modified'))
```